src/main/resources/static/scripts/dashboard.js

```diff
@@ -99,17 +98,51 @@
         } else if (!transactionType) {
             $("#typeTransactionError").text("Select type of transaction").show();
         }
+    }
+
+    function editTransactions() {
```

Comment:
I'd name it updateTransaction
update - because you update it, not editing
Transaction - because you update just one transaction